wg-lts/charter.md

```diff
@@ -0,0 +1,95 @@
+# WG Long Term Support
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+The Long Term Support Working Group (WG LTS) is organized with the goal of developing a better understanding what "Long Term Support" might mean for Kubernetes, those who support Kubernetes, and end users, as well as to investigate changes the Kubernetes project could make related to long term support. The working group will also determine feasibility, benefits, **costs**, and prerequisites of such changes.
+
+In the first phase of the working group, we will collect information related to the needs and wants regarding support periods from end-users and people supporting Kubernetes. This will be done by reaching out to users and cluster operators (e.g. via surveys), to better understand:
+
+* in-use versions (in the form of a user/vendor survey) and reasons for remaining on those versions
```

Comment:
For the survey, we have done a survey of our customers that is similar.
- The survey should cover that users may have multi clusters in multi versions. The cluster size and number of clusters also matter for upgradation. For instance, migration is another approach for multi-clusters. Some big clusters may be hard to upgrade due to complex configurations and some special dependencies(storage/networking) of pods. Some customers chose to drain old-version clusters like draining a node. `Drain old version cluster` means to migrate apps to new clusters when apps need an update.
-  How often does your cluster upgrade minor versions? How often does your cluster upgrade patch versions?

Reply:
It's a little early to describe all survey questions in this initial reformation PR. Some examples are good enough.

Admit that survey questions will impact the WG-LTS conclusion greatly, need to be designed and reviewed carefully.

for exmple, if the question is expected support period for each minor version, may receive answer "the longer the better". But if the question is can you accept only one longer supported minor version every year, the answer maybe "yes".

Existing facts will be easier to answer and expressed exactly. Expectation is not equal to real need, sometimes will be overstated.

Reply:
Agreed - probably don't need to expand too much on survey questions here. It's fine to list a few as examples, but there will likely be a significant refinement of them. We do have some data already from the regular Production Readiness Surveys.